test/specs/modules/Dimmer/Dimmer-test.js

```diff
@@ -0,0 +1,119 @@
+import _ from 'lodash'
+import faker from 'faker'
+import React from 'react'
+
+import Portal from 'src/addons/Portal/Portal'
+import Dimmer from 'src/modules/Dimmer/Dimmer'
+import DimmerDimmable from 'src/modules/Dimmer/DimmerDimmable'
+import * as common from 'test/specs/commonTests'
+import { sandbox } from 'test/utils'
+
+describe('Dimmer', () => {
+  common.isConformant(Dimmer)
+  common.hasSubComponents(Dimmer[DimmerDimmable])
+  common.hasUIClassName(Dimmer)
+
+  // TODO: Renders children / content
+  // common.rendersChildren()
```

Comment:
Yes, this test is broken. I think problem in `cloneElement` that I use for handling click. I'll try rework `click` handling without `stopPropagation`.